app/javascript/styles/_scatterPlot.scss

```diff
@@ -0,0 +1,98 @@
+
+.plot-container.plotly {
+  float: left;
+}
+
+.scatter-graph {
+  margin-left: 40px;
+  display: flex;
+}
+
+.col-md-6 .scatter-graph {
+  margin-left: 10px;
+}
+
+.scatter-legend {
+  float: right;
+  overflow: auto;
+  margin-top: 10px;
+  margin-left: 5px;
+}
+
+.scatter-legend-head {
+  margin: 5px 10px 10px 10px;
+}
+
+.scatter-legend-name {
+  font-weight: bold;
+}
+
+.stateful-link {
+  color: #4d72aa;
+  cursor: pointer;
+}
+
+.stateful-link.disabled {
+  color: rgb(197, 210, 227);
+  cursor: default;
+}
+
+.scatter-legend-row {
+  height: 30px;
+  cursor: pointer;
+  background-color: #FFF;
+  border: 1px solid #DDD;
+  border-radius: 3px;
+  margin-top: 3px;
+}
+
+/** Like hover state for React Select */
+.scatter-legend-row:hover {
+  border-color: #AAB;
+}
+
+.scatter-legend-row:hover {
+  border-color: #AAB;
+}
+
+.scatter-legend-icon {
+  width: 29px;
+  height: 29px;
+  margin: 0 10px 0 -1px;
+  float: left;
+  border-radius: 2px 0 0 2px
+}
+
+.scatter-legend-entry {
+  position: relative;
+  top: 5px;
+  overflow: scroll;
```

Comment:
Blocker: this rule is causing scrollbars to appear on individual legend entries.  I have a feeling my computer might be the only one, but that at least shows that it will probably happen to a subset of users.  Removing this rule seems to have no effect on functionality, and corrects the issue.

Example:
![Screen Shot 2021-11-15 at 2 17 21 PM](https://user-images.githubusercontent.com/729968/141842045-5ed67eab-2182-4094-8e71-1c60820e104b.png)

Reply:
Just pulled the branch -- I see the scrollbars too

Reply:
removing the overflow: scroll fixes it for me

Reply:
Fixed in e46bad5.  In my version of the same study, I couldn't see this, so my fix is somewhat blind.